internal/kokoro/test.sh

```diff
@@ -24,10 +24,24 @@ git clone . $GAX_HOME
 cd $GAX_HOME
 
 try3() { eval "$*" || eval "$*" || eval "$*"; }
-try3 go get -v -t ./...
 
-./internal/kokoro/vet.sh
+download_deps() {
+    if [[ `go version` == *"go1.11"* ]] || [[ `go version` == *"go1.12"* ]]; then
```

Comment:
```
if ! go help mod 2>/dev/null >/dev/null; then
```

(i used it here: https://github.com/GoogleCloudPlatform/golang-samples/pull/660/files#diff-8ddd3adc613dab276482e7911ac1f54aR73)

Reply:
Ack - I prefer the existing version. I reckon it's a bit easier to read and reason about, despite being verbose. Specifically that reasoning about whether the go version is a certain thing or not is a bit easier than reasoning about what the error code returned by `go help mod` is, and whether that's supported in earlier versions, and so on.

LMK if you feel strongly about this, though, and I'll reconsider.